### PR TITLE
Fix TravisCI config

### LINKED ISSUE #6670: Fix TravisCI Config Errors
**Describe the bug**
Viewing the config on TravisCI, users see a notification that some configuration values are incorrectly configured:

image

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to a PR
2. Click on the Travis checks
3. Click 'View Config'
4. See errors

**Expected behaviour**
TravisCI config should be valid

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by 12**
Resolves #6670

**Overall change:**
Fixes the errors and warnings in the TravisCI config.

**Code changes:**

- Updates `.travis.yml`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ cache:
 before_install:
   - sudo apt-get install -y libgconf-2-4
 jobs:
+  fast_finish: true
```
Comment (12) [inline .travis.yml:13]: `matrix` is an alias for `jobs`. [`fast_finish` is valid here](https://docs.travis-ci.com/user/customizing-the-build/#fast-finishing).

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # All Travis builds are a redundancy against Jenkins failures, except that we also use
 # this for running builds on PRs from forks, because we're OSS friendly. :)
 language: node_js
-matrix:
-  fast_finish: true
+os: linux
+dist: xenial
```
Comment (12) [inline .travis.yml:5]: Not strictly necessary to put these here explicitly as Travis defaults to them, but think it's better to be explicit.

**Review by jamesdonoh — APPROVED** [review]
Seems reasonable, can we get Travis to run on this branch to test it before merging?

**Comment by 12** [comment]
@jamesdonoh https://travis-ci.org/github/bbc/simorgh/builds/691779370

**Review by chris-hinds — APPROVED** [review]

**Comment by 12** [comment]
This config has caused TravisCI to be running in the wrong way, for a long time.

As `matrix` overwrote the `jobs` definition, there were no steps defined. As a result, Travis was running PRs with this configuration:

```
{
  "language": "node_js",
  "os": [
    "linux"
  ],
  "dist": "xenial",
  "cache": {
    "directories": [
      "~/.cache/Cypress"
    ]
  },
  "before_install": [
    "sudo apt-get install -y libgconf-2-4"
  ],
  "jobs": {
    "fast_finish": true
  },
  "notifications": {
    "email": [
      {
        "enabled": false
      }
    ]
  }
}
```

Note that there's no step definition in `jobs`. Travis was therefore, by default, running `npm ci` and `npm test`, and then exiting. When viewing a Travis build log, there would just be a large dump from the console.

By removing `matrix` and putting `fast_finish: true` into the `jobs` structure, the build config is now:

```
{
  "language": "node_js",
  "os": [
    "linux"
  ],
  "dist": "xenial",
  "cache": {
    "directories": [
      "~/.cache/Cypress"
    ]
  },
  "before_install": [
    "sudo apt-get install -y libgconf-2-4"
  ],
  "jobs": {
    "fast_finish": true,
    "include": [
      {
        "stage": "test",
        "if": "type = push OR (pull_request AND fork = true)",
        "name": "Unit Tests and storybook e2e",
        "install": [
          "npm ci"
        ],
        "script": [
          "npm run test"
        ]
      },
      {
        "stage": "test",
        "if": "type = push OR (pull_request AND fork = true)",
        "name": "Production CI tests",
        "install": [
          "npm ci"
        ],
        "addons": {
          "chrome": "stable"
        },
        "script": [
          "npm run build",
          "npm prune --production",
          "CYPRESS_SKIP_EU=true npm run test:ci"
        ]
      }
    ]
  },
  "notifications": {
    "email": [
      {
        "enabled": false
      }
    ]
  }
}
```

With this configuration, you can now see the two separate steps on the build page, and they run in parallel.

This has been incorrect for a long time, which is perhaps why we moved away from relying on Travis builds in the first place?

**Comment by jamesdonoh** [comment]
Good catch! Really glad we've got this working again.

**Pull request merged**